packages/server/graphql/types/JiraServerIntegration.ts

```diff
@@ -83,12 +91,8 @@ const JiraServerIntegration = new GraphQLObjectType<{teamId: string; userId: str
             'A list of projects to restrict the search to. format is cloudId:projectKey. If null, will search all'
         }
       },
-      resolve: async (
-        {teamId, userId},
-        // {first, queryString, isJQL, projectKeyFilters},
-        {first},
-        {authToken, dataLoader}: GQLContext
-      ) => {
+      resolve: async ({teamId, userId}, args: any, {authToken, dataLoader}) => {
+        const {first, queryString, isJQL, projectKeyFilters} = args as IssueArgs
```

Comment:
-1 If you type args, then do it in the function signature

Reply:
@Dschoordsch I took the approach from here 
https://github.com/ParabolInc/parabol/commit/072cdfcbe5cf1a5eed3a9c72ec1da51756d2b9fe#diff-00c4ea32101ae6a99d7e1beea2cfe1d6e9fa38b1a1c11d010558fa2c3db37b0dR111

Attempting to set type in function signature cause an error on `fields: () =>` above: 

<img width="931" alt="image" src="https://user-images.githubusercontent.com/466991/165511368-53c04995-cb1a-4d6f-b54e-6d92c61951d2.png">